cuda_redist_find_features/cmd/process_manifests.py

```diff
@@ -113,16 +52,25 @@ def process_manifests(
     # Create the version constraint
     version_constraint = VersionConstraint(min_version, max_version, version)
     # Parse and filter
-    refs = ManifestRef.from_ref(manifest_dir, version_constraint)
-    # Transform into manifests
-    manifest_redists: dict[Path, dict[str, manifest.Package]] = {
-        manifest_dir / f"redistrib_{ref.version}.json": ref.parse_manifest() for ref in refs
+    refs = NvidiaManifestRef.from_ref(manifest_dir, version_constraint)
+    # Transform into nvidia manifests
+    nvidia_manifests: dict[Path, NvidiaManifest] = {
+        manifest_dir / f"redistrib_{ref.version}.json": ref.parse() for ref in refs
```

Comment:
Parsing the Nvidia manifest is about as fast as it'll get in Python since there isn't any complicated data structure to deserialize or compute. (It's essentially just reading JSON from disk, which is reasonably fast in Python).